### DISCUSSION TITLE
would rust-skia like to publish under the name `skia-sys` and `skia`?

### DISCUSSION THREAD

**Discussion by coderedart** [rust-skia/rust-skia#905] [General]
I asked the owner of those crate names to publish a skiasharp based skia bindings crate. But they seem to be interested in giving those names to rust-skia instead. https://github.com/DiamondLovesYou/skia-rs/issues/2

I am actually fine with this too. I only tried because having those names would be nice. But i already had alternatives like ckia or rkia planned in case the owner didn't respond soon enough. But rust-skia being the defacto skia bidnings crate, i think it makes more sense for this project to publish under those names.

**Comment by coderedart** [comment]
@pragmatrix ping

**Reply by pragmatrix** [reply]
Thank you @coderedart for asking and @DiamondLovesYou for the offer, but the ship has already be named, changing names now would introduce confusion for everyone involved.

I'll let you decide what to do. 

I can imagine that these crates may be published with just a README.md and point to sharp-skia-rs?, tiny-skia, and skia-safe, explaining the differences for new users.
